### DISCUSSION TITLE
В чём дело?

### DISCUSSION THREAD

**Discussion by Ierichon** [vvip-68/GyverPanelWiFi#285] [Задавайте вопросы]
Скетч прошился, но в мониторе порта ничего не происходит? Что делать?
<spoiler>![Не работает](https://github.com/vvip-68/GyverPanelWiFi/assets/146264287/59f67daa-dddb-4d40-8c29-426fec77867c)</spoiler>

**Comment by vvip-68** [comment]
При нажатии на кнопку Reset микроконтроллера что-нибудь в мониторе порта отображается?
Если к плате контроллера что-нибудь подключено - отключите все. Чтобы только голая плата, шнурок USB в ней и более ничего. Перезапустите контроллер кнопкой Reset. Похоже что он даже не стартует.

**Reply by Ierichon** [reply]
При нажатии на кнопку RST появляются кракозябры
![2](https://github.com/vvip-68/GyverPanelWiFi/assets/146264287/9b18d771-fb76-4738-8d34-7f22f142ff19)

**Reply by vvip-68** [reply]
Ну, то есть связь с портом есть и он, вероятно, даже стартует.
Проверьте в настройках в a_def_hard.h не выключен ли у вас вывод данных в монитор порта для выбранного вами варианта конфигурации:
```
#define DEBUG_SERIAL 1
```
Вот тут должна быть 1

**Reply by Ierichon** [reply]
В доступных сетях Wi-Fi появляется сеть PanelAP без доступа, после того, как я к ней подключаюсь и пытаюсь настроить приложение на своем телефоне PanelSetupWiFi подключение PanelAP пропадает из доступных и более не появляется. Появляется лишь на некоторое время после очередной заливки скетча в NodeMCU. Причем Wled устанавливается и все работает прекрасно. Подскажите в чем дело, что я делаю не так, пятый день и ночь без сна, выучил ваш мануал от корки до корки и все равно ничего не получается

**Reply by vvip-68** [reply]
Если вы настроили IP - точка доступа и не будет больше появляться. Контроллер сразу будет подключаться к указанной сети, не создавая больше точку доступа

**Comment by Ierichon** [comment]
> Ну, то есть связь с портом есть и он, вероятно, даже стартует. Проверьте в настройках в a_def_hard.h не выключен ли у вас вывод данных в монитор порта для выбранного вами варианта конфигурации:
> 
> ```
> #define DEBUG_SERIAL 1
> ```
> 
> Вот тут должна быть 1

А у меня стоит "0" 
Сейчас попробую перешиться и отпишусь, но по моему я и "1" ставил в этом пункте и все было также. Сейчас, 1 минуту

**Reply by Ierichon** [reply]
Да, Вы просто меня выручили. Монитор порта запустился! УРА!
![Работает](https://github.com/vvip-68/GyverPanelWiFi/assets/146264287/df374741-f654-4ea8-b6ed-8dd65e206bf9)

**Reply by Ierichon** [reply]
Я же правильно понимаю, что если матрица весит на окне светодиодами на улицу и смотря на нее из квартиры подключение у меня происходит в правом нижнем углу по факту, то в скетче я должен прописывать левый нижний угол, верно же?

**Reply by vvip-68** [reply]
Ну, угол надо ставить тот, откуда вы на матрицу смотрите. Если смотрите из комнаты (и все надписи должны читаться в комнате) - значит один угол, если нужно чтобы читались с улицы - смотрите на матрицу с улицы и определяйте к какому из углов подведен сигнал. Логично же?

**Comment by Ierichon** [comment]
> Ну, угол надо ставить тот, откуда вы на матрицу смотрите. Если смотрите из комнаты (и все надписи должны читаться в комнате) - значит один угол, если нужно чтобы читались с улицы - смотрите на матрицу с улицы и определяйте к какому из углов подведен сигнал. Логично же?

Да, логично, а не подскажите, какое из трех apk, содержащихся в проекте рабочее, а то в каждом какие-то глюки

**Reply by vvip-68** [reply]
Все три рабочие и все три выполняют разные функции

**Reply by Ierichon** [reply]
Просто у меня одно только более-менее запустилось

**Reply by vvip-68** [reply]
А с остальными что?

**Reply by Ierichon** [reply]
В PanelSetup нет никаких режимов и эффектов, PanelPlayer только для рисования

**Reply by vvip-68** [reply]
В Setup и не должно быть никаких эффектов - это только для настройки матрицы.
И да - Player - только для рисования и игр.
И bingo!!! - для управления эффектами вам нужно третье приложение
